e2e/docker/testdata/TestComposeTemplate_commit_no_explorer.golden

```diff
@@ -30,7 +30,7 @@ services:
     labels:
       e2e: true
     container_name: mock_rollup
-    platform: linux/amd64
+    platform: linux/arm64
```

Comment:
this test is going to fail in CI since it is dependent on the OS ARCH that generated the golden file